km/km_mem.h

```diff
@@ -160,10 +182,10 @@ static inline km_gva_t gpa_to_upper_gva(uint64_t gpa)
 static inline int gva_to_memreg_idx(km_gva_t addr)
 {
    addr = gva_to_gpa(addr);   // adjust for gva in the top part of VA space
-   if (addr <= machine.guest_mid_physmem) {
-      return MEM_IDX(addr);
+   if (addr > machine.guest_mid_physmem) {
+      return machine.last_mem_idx - MEM_IDX(machine.guest_max_physmem - addr - 1);
    }
-   return machine.last_mem_idx - MEM_IDX(machine.guest_max_physmem - addr - 1);
+   return MEM_IDX(addr);
```

Comment:
This change seems to just rearrange code.
There is no logical difference.
Does this give some small performance advantage?